grafana/datasource.py

```diff
@@ -0,0 +1,52 @@
+import __builtin__
+import json
+import os
+import traceback
+from requests import post
+from grafana.connection import fread
+from grafana.connection import port_open
+from grafana.exceptions import FileNotFoundException
+
+HEADERS = {"Accept": "application/json",
+           "Content-Type": "application/json"
+           }
+
+''' Create Datasource '''
+
+
+def post_datasource(datasource_json):
+    NS = getattr(__builtin__, "NS")
```

Comment:
Not setting NS explicitly gives a pep8 error, which is why I had to reset the NS.
I will remove it, but please ignore the errors in pep8 for NS (undefined name 'NS')